COCOAPODS.md

```diff
@@ -1,150 +1,343 @@
+  Xcode build steps manually.
+ * When building from Xcode, you can use CocoaPods libraries without writing
+ `.def` files manually and setting `cinterop` tool parameters. In this case, all required parameters can be
+ obtained from the Xcode project configured by CocoaPods.
+ 
+You can add dependencies between:
+* [A Kotlin Pod and an Xcode project with one target](#add-a-dependency-between-a-kotlin-pod-and-xcode-project-with-one-target)
+* [A Kotlin Pod and an Xcode project with several targets](#add-a-dependency-between-a-kotlin-pod-with-an-xcode-project-with-several-targets)
+* [Several Kotlin Pods and an Xcode project](#add-dependencies-between-multiple-kotlin-pods-and-an-xcode-project)
 
-1. Configure a Gradle project: apply the `org.jetbrains.kotlin.native.cocoapods` plugin, add
-and configure the targets, and specify the required podspec fields.
+> To correctly import the dependencies into the Kotlin/Native module, the
+`Podfile` must contain either [`use_modular_headers!`](https://guides.cocoapods.org/syntax/podfile.html#use_modular_headers_bang)
+or [`use_frameworks!`](https://guides.cocoapods.org/syntax/podfile.html#use_frameworks_bang)
+directive.
+{:.note}
 
-2. Run the `podspec` task. The podspec file described above will be generated.
+### Add a dependency between a Kotlin Pod and Xcode project with one target
 
-    > In order to avoid compatibility issues during an Xcode build, the plugin requires using
-    a [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). 
-    To generate the wrapper automatically during execution of the `podspec` task,
-    run it with the parameter `-Pkotlin.native.cocoapods.generate.wrapper=true`.
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Add the path to your `Podfile` with `podfile = projectDir.resolve("..")` to `build.gradle.kts` (`build.gradle`) 
+of your Kotlin project.
 
-3. Add a reference to the generated podspec in a Podfile of the Xcode project.
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+        
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("AFNetworking", "~> 3.2.1")
+            podfile = projectDir.resolve("../ios-app/Podfile")
+        }
+    }
+    ```
+    
+    </div>
 
-    <div class="sample" markdown="1" theme="idea" mode="ruby">
+3. Add the name and path of the Kotlin Pod you want to include in the Xcode project to `Podfile`.
 
+    <div class="sample" markdown="1" theme="idea" mode="ruby" data-highlight-only>
+    
     ```ruby
-    target 'my-ios-app' do
-        pod 'my_kotlin_library', :path => 'path/to/my-kotlin-library'
+    use_frameworks!
+    
+    platform :ios, '9.0'
+    
+    target 'ios-app' do
+            pod 'kotlin_library', :path => '../kotlin-library'
     end
     ```
+    
+    </div>
 
+4. Re-import the project.
```

Comment:
May be add that we call `pod install` for the user's Podfile under the hood at this step? Probably a user doesn't expect this so it's better to warn him.

Reply:
In point 2 we decided to add some explanation about `podfile` setting